_posts/2016/2016-05-12-offsite-consulting.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "Consulting offsite meeting"
+date: "2016-05-18 16:03:17"
+image: blog-header/consulting-offsite.jpg
+categories: events
+authors: ["Pavlo", "Tobias"]
+---
+
+On 1st of May 2016, the ePages Consulting team set off on a journey to Bad Sulzburg, Germany. After a six-hour drive from Jena, we were welcomed by the nice staff of the Waldhotel, which is situated in a quiet spot of the Black Forest. A good place for us to reflect on what works well in the team and what could be improved.
+
+{% image blog/blog-consulting-offsite-1.jpg %}{% endimage %}
+
+The purpose of this particular trip was to talk about opportunities and challenges faced by our team over time as well as to come up with strategic plans for the future.
+
+## Team building activities
+
+To strengthen the team spirit and escape from office life, we decided to go hiking and enjoy the exceptional diversity and grandiose views of the Black Forest landscape.
+
+{% image blog/blog-consulting-offsite-2.jpg %}{% endimage %}
+
+On the next day after countless meetings we played Kubb to reduce stress and to draw fresh inspiration. Kubb is an exciting team game from Sweden with a very long tradition. The goal of each team is to overturn the wooden figures of the counterparty by throwing wooden sticks. The king with the red crown in the middle of the field must be taken last.
```

Comment:
On the next day, after countless meetings, we played Kubb to reduce stress and to draw fresh inspiration.